### PR TITLE
Update the Now Online page, add the Company column and edit customer session lifetime

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request updates the Now Online page, adds the Company column and edits customer session lifetime

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/customers/now-online.html

### CODE REVIEW

File: src/customers/now-online.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Now Online
 ---
 
-The "Now Online" option on the Customers menu lists all customers and visitors who are currently online in your store. The interval of time that customers are shown as currently online is set in the configuration, and determines how long the customer’s activity is visible from the Admin. By default, the interval is fifteen minutes.
+The **Now Online** option on the Customers menu lists all customers and visitors who are currently online in your store. The interval of time that customers are shown as currently online is set in the configuration, and determines how long the customer’s activity is visible from the Admin. By default, the interval is fifteen minutes. The session ends if the keyboard is not used during this time and customers must login into their accounts again to continue shopping. It is important to note that the contents of the carts are saved for later access.
```
Comment (jcalcaben) [inline src/customers/now-online.md:5]:
```suggestion
The **Now Online** option on the Customers menu lists all customers and visitors who are currently online in your store. The interval of time that customers are shown as currently online is set in the configuration, and determines how long the customer’s activity is visible from the Admin. By default, the interval is fifteen minutes. The session ends if the keyboard is not used during this time and customers must sign into their accounts again to continue shopping. It is important to note that the contents of the carts are saved for later access.
```

**Review by jcalcaben — CHANGES_REQUESTED** [review]

File: src/customers/now-online.md (hunk 1 above)
Comment (ZhovnirMaria) [reply src/customers/now-online.md]: Hi, @jcalcaben , thank you!
Done ✅

**Comment by jcalcaben** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@jcalcaben the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1226`.

**Pull request merged**